OfferSpace.BL/Models/RequestCustomer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OfferSpace.BL.Models
+{
+    public class RequestCustomer : IEntity<long>
+    {
+        [Key]
+        public long Id { get; set; }
+
+        //[ForeignKey("RequestId")]
+        public long? RequestId { get; set; }
+        public Request Request { get; set; }
+
+        public long? ExecutorId { get; set; }
+        public Executor Executor { get; set; }
+
+        public DateTime ScheduleOn { get; set; }
+
+        public bool MarkAsDeleted { get; set; }
+
+        //[ForeignKey("RequestCustomerDiscusion")]
+        public long? RequestCustomerDiscusionId { get; set; }
```

Comment:
Please remove it. RequestCustomer has the list of discussion and can't have only one reference on discussion, please use List<RequestCustomerDiscussion> instead.